trieste/utils/pareto.py

```diff
@@ -71,27 +74,37 @@ class Pareto:
     A :class:`Pareto` Construct a Pareto set.
     Stores a Pareto set and calculates the cell bounds covering the non-dominated region.
     The latter is needed for certain multiobjective acquisition functions.
+
+    For hypervolume-based multiobjective optimisation with n>2 objectives, this class
+    implement branch and bound procedure algorithm. a divide and conquer method introduced
+    in :cite:`Couckuyt2012`.
     """
 
-    def __init__(self, observations: TensorType):
+    def __init__(self, observations: TensorType, *, jitter: float = DEFAULTS.JITTER):
         """
-        :param observations: The observations for all objectives, with shape [N, 2].
+        :param observations: The observations for all objectives, with shape [N, D].
         :raise ValueError (or InvalidArgumentError): If ``observations`` has an invalid shape.
         """
-        tf.debugging.assert_shapes([(observations, [None, 2])])
+        tf.debugging.assert_rank(observations, 2)
```

Comment:
(Do we still want to assert that `shape[-1] >= 2`?)

Reply:
For me personally, it might be nice to have that assertion.